### ISSUE TITLE
A11y_AzureDataStudioSQL_Add Database reference_System Database_ScreenReader:Nvda does not announces required when user navigates on Database name edit box with the help of tab key 

### ISSUE THREAD

**Issue by Yash14j** [microsoft/azuredatastudio#14360]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG3.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.27.0-insider
Windows Version: Windows10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Few databases will appear under your connection then right click on any database
5. Tab till "Create project from database" and hit enter.
6. New database project will get created under "Projects" tab and tab till "Database references"
7. Right Click and hit enter on "Add Database references".
8. Select "System Database" radio button.
9. Start the nvda
10. Navigate to Database name edit box and check the result

### Actual:
Nvda does not announces required when user navigates on Database name edit box with the help of tab key .
Same issue is observed with Jaws

Similar issue is observed in below scenarios:

Add Database reference- Data tier application
Add Database reference- Projects

### Expected:
Nvda should announce  required when user navigates on Database name edit box with the help of tab key .

### User Impact:
Screenreader user will not able to know that it is a required field. .
